src/tests/FunctionalTests/Android/Device_Emulator/gRPC/grpc-dotnet/test/Shared/HttpEventSourceListener.cs

```diff
@@ -0,0 +1,120 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
```

Comment:
we also need to add a third party notices file with this license notice in the root of the folder, see e.g. src/tests/JIT/Performance/CodeQuality/V8/Crypto/THIRD-PARTY-NOTICES